### ISSUE TITLE
Support for Folia

### ISSUE THREAD

**Issue by Tibarica** [QarthO/qXP-Boosts#1]
Pls add support for folia! Thx

**Comment by QarthO** [comment]
Sure, I'll look into adding it tomorrow. Never worked with Folia before, but should be simple for xpboosts

**Comment by QarthO** [comment]
@Tibarica Do you want to try this to see if it works for you
https://github.com/QarthO/qXP-Boosts/actions/runs/10787973476

**Comment by Tibarica** [comment]
It works! Thank you!. Can you also add a lang file?

**Comment by QarthO** [comment]
give them the permission for the boost. 
<img width="945" alt="image" src="https://github.com/user-attachments/assets/c58bc95b-9756-4ce2-adfb-219aababdda5">

**Comment by Tibarica** [comment]
Yeah, thank you very much!. Let me know if you can add a lang_config file so we can edit the messages :P 
Thank you again!

**Comment by QarthO** [comment]
Custom language support foundation is already in, I just need to implement it. I'll work on that this week and get that pushed to the live version
